### PR TITLE
docs(restore): former branch was messed up

### PR DESCRIPTION
**Pull request by ff6347**
Moved all docs files to new branch closes #191

### CODE REVIEW

**Review by dnsos — APPROVED** [review]
Thanks for these docs (and creating this new PR), @ff6347.

I fixed a few more typos and tried running the JS as described. The test record is added 🎉

**Comment by dnsos** [comment]
@Esshahn if you wanna have a look at the new docs, go ahead. If not, let me know if we can merge this.

**Comment by Esshahn** [comment]
I'll review them asap

**Review by dnsos — APPROVED** [review]

**Comment by Esshahn** [comment]
I finished my docs review. Can be merged now.

**Review by Esshahn — APPROVED** [review]

File: pages/docs/ttn-sensor.mdx [hunk 1]
```diff
@@ -31,25 +30,25 @@ Dort angekommen, kannst du einen neuen Webook für deine Applikation einrichten.
...
 
-Unter den Endpoint Settings muss nun der API URL konfiguriert werden. In unserem Falle nutzen wir als **Base URL** des Endpoints die RESTful API von Stadtpuls: 
+Unter den Endpoint-Settings muss nun die API URL konfiguriert werden. In unserem Falle nutzen wir als **Base URL** des Endpoints die RESTful API von Stadtpuls: 
 
 ```plain
 https://api.stadtpuls.com
 ```
 
-Wir übergeben der API jedoch noch eine zusätzliche Information – **Additional headers** mit dem Namen *authorization* im Header: nämlich deinen Authorizierungs-Token den du dir bereits angelegt hast.
+Wir übergeben der API jedoch noch eine zusätzliche Information – **Additional headers** mit dem Namen *authorization* im Header: nämlich deinen Authorisierungs-Token, den du dir bereits angelegt hast.
```
Comment (ff6347) [inline pages/docs/ttn-sensor.mdx:43]: Ich glaube es müsste eigentlich "Authentifizierungs-Token" sein https://translate.google.com/?sl=auto&tl=de&text=HTTP%20authentication&op=translate

**Review by ff6347 — COMMENTED** [review]
@Esshahn Nice. Thank you. One thing I noticed is that we sometimes have double quotes and sometimes single quotes. Which is a little annoying. The other thing is the "Authentifizierungs-Token" comment

File: pages/docs/ttn-sensor.mdx (hunk 1 above)
Comment (Esshahn) [reply pages/docs/ttn-sensor.mdx:43]: agree to both. Please go ahead and change it according to your suggestion. Good find!

**Pull request merged**